fiaas_mast/common.py

```diff
@@ -53,10 +52,9 @@ def make_safe_name(name):
 
 
 def select_models():
```

Comment:
This function is no longer needed, as there is only one set of models to use.

Reply:
This method also checks for the model to exist in the cluster, which I think it's still a useful thing to do and keep in a single place. What do you think?

Reply:
Good point, but then it should change name to better reflect what it does.

Reply:
makes sense 👍  `verify_and_get_model`, `check_model`? Any suggestion is more than welcome 😄